src/test/firestore/indexes.spec.ts

```diff
@@ -117,6 +118,29 @@ describe("IndexNameParsing", () => {
       "/projects/myproject/databases/(default)/collectionGroups/collection/fields/abc123/";
     expect(util.parseFieldName(name)).to.eql({
       projectId: "myproject",
+      databaseId: "(default)",
+      collectionGroupId: "collection",
+      fieldPath: "abc123",
+    });
+  });
+
+  it("should parse an index name from a named database correctly", () => {
+    const name =
+      "/projects/myproject/databases/named-db/collectionGroups/collection/indexes/abc123/";
+    expect(util.parseIndexName(name)).to.eql({
```

Comment:
Okay, there are tests for `util`, but they're in the `indexes.spec.ts`  file? That's... strange. I'd expect tests for `util` functions to be in `util.spec.ts`... maybe not worth fixing in this CL (because it'll be a lot of extra changes to sort though), but maybe follow up to address that.

Reply:
good idea, this isn't a huge change - moved